ts/nni_manager/core/nnimanager.ts

```diff
@@ -458,6 +467,9 @@ class NNIManager implements Manager {
         } else if (platform === 'adl') {
             const module_ = await import('../training_service/kubernetes/adl/adlTrainingService');
             return new module_.AdlTrainingService();
+        } else if (platform == 'retiarii_local') {
```

Comment:
I don’t think it should be a separate training service.
We won’t want to copy every training service.